tests/test_http_ext_auth.py

```diff
@@ -350,29 +350,29 @@ class TestHttpExtAuth(tb.ExtAuthTestCase):
 
         CONFIGURE CURRENT DATABASE
         INSERT ext::auth::GitHubOAuthProvider {{
-            secret := <str>'{GITHUB_SECRET}',
-            client_id := <str>'{uuid.uuid4()}'
+            secret := '{GITHUB_SECRET}',
+            client_id := '{uuid.uuid4()}',
         }};
 
         CONFIGURE CURRENT DATABASE
         INSERT ext::auth::GoogleOAuthProvider {{
-            secret := <str>'{GOOGLE_SECRET}',
-            client_id := <str>'{uuid.uuid4()}'
+            secret := '{GOOGLE_SECRET}',
+            client_id := '{uuid.uuid4()}',
         }};
 
         CONFIGURE CURRENT DATABASE
         INSERT ext::auth::AzureOAuthProvider {{
-            secret := <str>'{AZURE_SECRET}',
-            client_id := <str>'{uuid.uuid4()}'
+            secret := '{AZURE_SECRET}',
+            client_id := '{uuid.uuid4()}',
+            additional_scope := 'offline_access',
```

Comment:
Should we name this `additional_scopes`? (I assume we allow multiple scopes?)

Reply:
I went back and forth on this: is the plural of scope really scopes? Or is it a single scope that is now bigger (increasing the scope of something). The parameter in OAuth2 is called `scope`, so this _does_ feel kinda right.

And, indeed, this is a space-separated list of scope strings.

Reply:
I think it should be "scopes" -- as in the purpose of this is to extend the information we request and potentially add more information "bits" -- so using a plural "scopes" here seems like a better option to me.